docs/proposals/image-signing.md

```diff
@@ -0,0 +1,129 @@
+# Image Signing
+
+## Abstract
+
+This proposal describes the process of verifying the signatures of Docker Images and
+enforce enforce policy that will prevent the image from running in the cluster if the
+image signature is not trusted.
+
+## Use-cases
+
+* As an admin, I want to be able to add trusted GPG public keys to my cluster and determine through policy
+  the keys that are required to run certain images
+* As an user, I want to be able to see what images are signed and if the signature is trusted
+
+## Overview
+
+### Signing Images with Skopeo
+
+The current signing mechanism for Docker image is to use the
+[Skopeo](https://github.com/projectatomic/skopeo/blob/master/docs/skopeo.1.md) tool which
+will sign the Docker image locally, using the GPG key and attach the signature to the
+Docker Image. The Docker image is then pushed to a Docker Registry with enabled signature
```

Comment:
* the signature can be either stored locally as a file "blob" ("docker" transport type) or posted to the openshift API ("atomic" transport type). See [signature API](https://docs.openshift.org/latest/rest_api/openshift_v1.html#create-a-imagesignature).
* The signature is a detached object that can be stored and served independent of the associated image.